forms-web-app/tests/unit/validators/appellant-submission/site-location.test.js

```diff
@@ -368,6 +368,38 @@ describe('validators/appellant-submission/site-location', () => {
           expect(result.errors[0].msg).toEqual('Postcode must be 8 characters or fewer');
         },
       },
+      {
+        title: 'invalid - site-postcode shoudl begin with a letter',
+        given: () => ({
+          body: {
+            'site-address-line-one': '1 Taylor Road',
+            'site-address-line-two': 'Clifton',
+            'site-town-city': 'Bristol',
+            'site-county': 'South Glos',
+            'site-postcode': '111aaa',
+          },
+        }),
+        expected: (result) => {
+          expect(result.errors).toHaveLength(1);
+          expect(result.errors[0].msg).toEqual('Postcodes should begin with a letter');
+        },
+      },
+      {
+        title: "invalid - site-postcode can't be only letters",
+        given: () => ({
+          body: {
+            'site-address-line-one': '1 Taylor Road',
+            'site-address-line-two': 'Clifton',
+            'site-town-city': 'Bristol',
+            'site-county': 'South Glos',
+            'site-postcode': 'aaaaaaa',
+          },
+        }),
+        expected: (result) => {
+          expect(result.errors).toHaveLength(1);
+          expect(result.errors[0].msg).toEqual("Postcodes can't be all letters");
```

Comment:
are these error messages approved by Marc & Daisy?

Reply:
It was on the feature file so I guess. And I hope because I had to add this behavior...

Reply:
yeah we need to be a bit careful here.

The feature file is supposed to be 'logical'. If there's a reason a thing should fail, we want to state what that reason is.
It is -not- stating the text the user should see